packages/block-editor/src/components/block-list/style.scss

```diff
@@ -601,30 +601,11 @@
 	display: none;
 }
 
-
-// Add side padding for the canvas, currently edit-post-visual-editor.
-// The purpose of this padding is to ensure that on small viewports, there is
-// room for the block border that sits 14px ($block-padding) offset from the
-// block footprint.
-// These paddings and margins are removed from the BlockPreview component's style
-// Any change need to be reflected there.
+// The editor-styles.scss file adds 10px padding all around the canvas, to ensure blocks aren't edge to edge on mobile.
+// For full-wide blocks, we compensate for these 10px.
 .block-editor-block-list__layout.is-root-container {
```

Comment:
Should we move these padding resets to the same file where the padding is added?

Reply:
As of https://github.com/WordPress/gutenberg/pull/22213/commits/44e945f023f1145e5588f8becc5dd0fcdb06e2a3 that would move it to `editor-styles.scss`. Happy to move it there if you feel it's appropriate? I personally like it because it keeps it together, but it feels... wrong?

Reply:
I wanted to be able to do this:

```
body {
	font-family: $editor-font;
	font-size: $editor-font-size;
	line-height: $editor-line-height;
	color: $dark-gray-primary;
	padding: 10px;

	// For full-wide blocks, we compensate for these 10px.
	> .wp-block[data-align="full"] {
		margin-left: -10px;
		margin-right: -10px;
	}
}
```

but body maps to editor-styles-wrapper, and `block-editor__typewriter` is the immediate child of that. So we have to target `.is-root-container`, which we can, but it's not quite as nice. I.e. this works:

```
body {
	font-family: $editor-font;
	font-size: $editor-font-size;
	line-height: $editor-line-height;
	color: $dark-gray-primary;
	padding: 10px;
}

// For full-wide blocks, we compensate for these 10px.
.is-root-container > .wp-block[data-align="full"] {
	margin-left: -10px;
	margin-right: -10px;
}
```

Reply:
I'd use the full selector `.block-editor-block-list__layout.is-root-container` but I think that's a good approach personally.